src/pages/Main/Main.js

```diff
@@ -0,0 +1,58 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { wordsActions } from '../../actions/wordsActions';
+import { getHundredWords } from '../../api/getHundredWords';
+import PagesForm from '../../components/Forms/PagesForm/PagesFrom';
+import SearchForm from '../../components/Forms/SearchForm/SearchForm';
+import WordsOnMainPage from '../../components/WordsOnMainPage/WordsOnMain';
+import { DispatchWordsContext, StateWordsContext } from '../../context/WordsContext';
+import { Tab } from '../../components/Tab/Tab';
+import './main.scss';
+import { Table } from '../../components/Tabs/Table';
+import { Subscription } from '../../components/Tabs/Subsccription';
+
+
+const Main = () => {
+    const [tabsSwitch, setTabsSwitch] = useState(1);
+    const store = useContext(StateWordsContext);
+    const dispatch = useContext(DispatchWordsContext);
+    const {firstHundredWords} = store;
+    useEffect(() => {
+        getHundredWords().then(data => 
+            dispatch({
+                type: wordsActions.FETCH_FIRST_HUNDRED_WORDS,
+                payload: data.details
+            })
+        )
+    }, [dispatch]);
+    const setIndexForTab = (index) => {
+        setTabsSwitch(index)
+    }
+   
+    return  (
+        <div className="main-page">
+            <div className="main-page__item">
+                <SearchForm />
+                <WordsOnMainPage words={firstHundredWords} />
+                <PagesForm />
+            </div>
+            <div className="main-page__item tabs-box">
+                <div className="tabs-box__links">
```

Comment:
It terms of reusable components, we do need to work with them as with black-box, so our Tabs component should look like I described bellow:  

`<Tabs activeTabIndex={value} onChange={handleChange} tabList={[
  {
    label: 'Table',
    content: <TableComponent />
  },
  {
    label: 'Subscription',
    content: <Subscription />
  }
  ]}
/>`